src/DashboardService.Mappers/Models/GroupInfoMapper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using LT.DigitalOffice.DashboardService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.DashboardService.Models.Db;
+using LT.DigitalOffice.DashboardService.Models.Dto.Models;
+using LT.DigitalOffice.Kernel.Extensions;
+using Microsoft.AspNetCore.Http;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Mappers.Models;
+
+public class GroupInfoMapper : IGroupInfoMapper
+{
+  private readonly IHttpContextAccessor _httpContextAccessor;
+
+  public GroupInfoMapper(IHttpContextAccessor httpContextAccessor)
+  {
+    _httpContextAccessor = httpContextAccessor;
+  }
+
+  public GroupInfo Map(DbGroup dbGroup)
+  {
+    return dbGroup is null
+      ? null
+      : new()
+      {
+        Id = dbGroup.Id,
+        BoardId = dbGroup.BoardId,
+        Name = dbGroup.Name,
+        IsActive = dbGroup.IsActive,
+        CreatedBy = _httpContextAccessor.HttpContext.GetUserId(),
+        CreatedAtUtc = DateTime.UtcNow,
+        ModifiedBy = _httpContextAccessor.HttpContext.GetUserId(),
```

Comment:
you can place result of _httpContextAccessor.HttpContext.GetUserId() to variable and use it twice instead of double call of method

Reply:
++